# This file is distributed under the same license as the Django package.
#
# Translators:
# Jannis Leidel <jannis@leidel.info>, 2011
# Shinya Okano <tokibito@gmail.com>, 2012
msgid ""
msgstr ""
"Project-Id-Version: django-core\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2013-05-02 16:18+0200\n"
"PO-Revision-Date: 2013-04-26 14:19+0000\n"
"Last-Translator: claudep <claude@2xlibre.net>\n"
"Language-Team: Japanese (http://www.transifex.com/projects/p/django/language/"
"ja/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ja\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: models.py:130
msgid "python model class name"
msgstr "Python モデルクラス名"

#: models.py:134
msgid "content type"
msgstr "コンテンツタイプ"

#: models.py:135
msgid "content types"
msgstr "コンテンツタイプ"

#: views.py:17
#, python-format
msgid "Content type %(ct_id)s object has no associated model"
msgstr ""
"コンテンツタイプ %(ct_id)s のオブジェクトは、関連付けられたモデルを持っていま"
"せん"

#: views.py:21
#, python-format
msgid "Content type %(ct_id)s object %(obj_id)s doesn't exist"
msgstr "コンテンツタイプ %(ct_id)s のオブジェクト %(obj_id)s は存在しません"

#: views.py:27
#, python-format
msgid "%(ct_name)s objects don't have a get_absolute_url() method"
msgstr ""
"%(ct_name)s のオブジェクトは get_absolute_url() メソッドを持っていません"
